### DISCUSSION TITLE
Temp Error on temp change

### DISCUSSION THREAD

**Discussion by nor1bie** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2063] [Ideas]
Not sure if anyone else has seen this issue.
With the latest firmware (pre-built BIGTREE_TFT35_V3.0_E3.27.x  Jul 7. 2021 or somewhere around) for the TFT35 E3 V3, I see fatal errors when trying to print a Temp Tower with embedded temperature changes. Initially, I have a start temp of 205C set in gcode. After leveling, the gcode instructs to change to 220C, however it continues to print @ 205C (this maybe a gcode issue). After the 220C section (which is at 205C) encounter's the change to 215C a temp error (I believe timeout) is thrown. This is a fatal error and the printer halts. 
Using the same code feeding it through onboard SD slot, it seems to work without the fatal error. But, the temp change is still ignored and I have to change it manually throughout the print. 
My guess would be that on either side something with the temperature algorithm doesn't work correctly. 

The reason I start here is simply because it doesn't throw the error when printing through onboard sd.  

I have to further investigate the problem with not changing the temperature, but think there are too many cooks involved and mess up the dish. In the short time, since October 2020, I am using Marlin / TFT35, I have been scratching my head so much, that I became bald.
Many times I made changes on one end whilst the other end somehow managed to do something else. At least it did something not logical to me. It is very difficult to follow what device is doing what and how they interact, meaning not knowing what side did cause the issue, leading to mostly something that does only work half. Once I got to a point where things seemed to work in harmony, I upgraded the more better, more gooder firmware and the entire thing starts over again.
Most of the time you can't use the known working config file, someone decided to make the incompatible. 

Would be interesting to know if anyone else has a similar experience, since I could not find anything on Google pertaining to the temperature errors.

**Comment by kisslorand** [comment]
I suggest to move this to the "issues" section, it is more likely to be seen by others. There's a button on the right to "Create issue from discussion".

**Reply by nor1bie** [reply]
TNX, I'll move it to issues.
Cheers
